src/nrnpython/CMakeLists.txt

```diff
@@ -83,8 +83,10 @@ endif()
 
 if(NRN_ENABLE_MEMACS)
   set(BUILD_MEMACS_TRUE "")
+  unset(WITHOUT_MEMACS)
 else()
   set(BUILD_MEMACS_TRUE "#")
+  set(WITHOUT_MEMACS 1)
```

Comment:
Do you meant `add_definitions` / `set_target_properties` here? (`set` just set CMake scope variable)

Reply:
IIRC this ends up in nrnconf. Will check tomorrow.